cross/arm32_ci_script.sh

```diff
@@ -189,13 +189,13 @@ function cross_build_corefx_with_docker {
         # TODO: For arm, we are going to embed RootFS inside Docker image.
         case $__linuxCodeName in
         trusty)
-            __dockerImage=" microsoft/dotnet-buildtools-prereqs:ubuntu1404_cross_prereqs_v3"
+            __dockerImage=" microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20172211042239"
```

Comment:
@gkhanna79 - depends on your view of friendly 😄.  The first half is basically the same.  The second half consists of the commit sha of the Dockerfile the image was produced from and the timestamp of when it was built.  This will help us know exactly how the image was produced.  Prior to this we had no way of tracking this information.  This change was introduced to better support forkable engineering.